src/main/java/com/google/edith/interfaces/LoginInterface.java

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.edith.interfaces;
+
+import javax.servlet.http.HttpServletRequest;
+
+/** Operations for handling user authentication. */
+public interface LoginInterface {
+  public String createJsonOfUserInfo();
```

Comment:
Nit: You can remove the public keyword for methods in an interface.

Reply:
Thanks.
Changes made.